kickstarts/partials/packages/includes.ks.erb

```diff
@@ -91,6 +91,8 @@ npm
 <% if @target == "ovirt" %>
 ovirt-guest-agent
 glusterfs-fuse
+<% elsif @target == "libvirt" %>
+qemu-guest-agent
```

Comment:
Minor but in a followup PR can we sort this and use a case statement? cc @simaishi